charts/ratify/values.yaml

```diff
@@ -16,8 +17,8 @@ resources:
     cpu: 1000m
     memory: 512Mi
   requests:
-    cpu: 600m
-    memory: 512Mi
+    cpu: 200m
```

Comment:
I primarily did this because our e2e tests were failing. The github runners don't have enough node capacity to handle the Dapr + redis + 2 ratify replicas.

I will revert this change and override value in the test.